File: src/main/java/com/fatec/destino/repository/pacote/PacoteFotoRepository.java
package com.fatec.destino.repository.pacote;

import com.fatec.destino.model.pacote.pacoteFoto.PacoteFoto;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PacoteFotoRepository extends JpaRepository<PacoteFoto, Long> {
}

File: src/main/kotlin/com/fatec/destino/repository/pacote/PacoteFotoRepository/PacoteFotoRepository.kt
package com.fatec.destino.repository.pacote.PacoteFotoRepository

import com.fatec.destino.model.pacote.pacoteFoto.PacoteFoto
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PacoteFotoRepository : JpaRepository<PacoteFoto, Long> {
}